src/styles/index.scss

```diff
@@ -1,3 +1,113 @@
-body {
-  margin: 0;
+$font-type: "Roboto", sans-serif;
+$font-main-color: #060b35;
+$font-secondary-color: #616070;
+$button-color: #00acdc;
+$card-border-color: #f3f3f3;
```

Comment:
you can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`)

Reply:
OK I will, thanks!